blog/2021-12-13-maintainers-interview.md

```diff
@@ -0,0 +1,120 @@
+
+**Paweł:** My adventure with computer science also started pretty early. When I got my first PC, I was 8 years old. Initially, I played games on it, but after a while, I was interested in doing more than that. When I was in primary school, I started to write some basic applications in Delphi and created my first HTML website. Obviously, the websites followed the trends at that time—they were loaded with GIFs and tons of text scrolling in different directions (remember the “MARQUEE” HTML tag?).
+
+When I was 13, I began working as a freelancer in the web development area. Before getting a bachelor’s degree in computer science, I joined my first IT company as a UI developer. More than 4 years ago, I transformed from a front-end guy to a full-stack cloud developer, with JavaScript, Go, and Kubernetes. I was involved in building a project from the CNCF landscape, and over time I developed less and less front-end and more back-end stuff. In the meantime, I was a Scrum Master and was involved in promoting that previous project at conferences, meetups, and different events. Same as Mateusz, I joined Capact over a year ago to build an ambitious, cloud-native project from scratch.
+
+## The main technologies used in Capact are Go and Kubernetes. Why such a choice in this case?
+
+**Paweł:** Both Go and Kubernetes are becoming more and more popular.
+
+In the cloud-computing world, choosing the proper set of technologies is very crucial to ensure the final application is scalable, performant, and resilient. Kubernetes helps us orchestrate the deployed microservices, and make sure they operate without any issue, on any cloud provider, or even bare metal.
+
+Go, on the other hand, is a great fit for such projects—its simplicity enables us to write such back-end services efficiently. They are fast to compile and consume much fewer resources than alternatives written in Java or JavaScript/TypeScript.
+
+**Mateusz:** It’s worth highlighting that the Go community already created a lot of powerful libraries. You don’t need to solve cloud-specific problems once again, and can focus on solving your domain/business problems instead.
+
+As mentioned, Go is easy in terms of memory consumption, which is important in cloud environments. Of course, it doesn't come without a downside; the abstraction level in Go is quite low, but the upcoming Go 1.18 will be a significant release, e.g. support for generic.
+
+## Can you give us more background on what Capact is?
+
+**Paweł:** Capact is an open-source framework for running arbitrary workflows. Initially, we are focused on day-one and day-two workflows for deploying, and managing applications and cloud-native infrastructure.
```

Comment:
Capact is an open-source framework for running arbitrary workflows. Currently, we are focusing on day-one and day-two workflows for deploying, and managing applications and cloud-native infrastructure.